jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/UploadObjectAction.kt

```diff
@@ -70,9 +70,9 @@ fun uploadObjects(project: Project, treeTable: S3TreeTable, files: List<VirtualF
                 }
             }
 
-            S3Telemetry.uploadObjects(project, Result.Succeeded, files.size.toDouble())
+            S3Telemetry.uploadObjects(project, Result.Succeeded)
```

Comment:
Why were these changed?

Reply:
Their definition was changed.

Reply:
File files size should map to value, so you just need to switch to named args